roles/gaia/tasks/main.yml

```diff
@@ -188,14 +188,20 @@
 #     path: "{{gaiad_home}}/config/genesis.json"
 #     get_checksum: false
 #   register: genesis_file_exists
-- name: copy genesis.json
+- name: copy genesis.json.gz
```

Comment:
ideally the ansible should:
1. check if the file is gzipped or not
2. if not zipped, it should first zip it
3. then it should cp over
4. then if zipped, it should unzip